Jenkinsfile-e2e

```diff
@@ -27,14 +27,18 @@ node {
         ]) {
             cleanWs() // Clean the workspace
 
+            def hashToCheckout = params.BRANCH == '' ? Simorgh.getLiveCommitHash() : params.BRANCH
+
             docker.image(Simorgh.nodeDockerImage()).inside('--ipc host') {
                 try {
-                    Simorgh.checkout("${params.BRANCH}", 'simorgh')
+                    Simorgh.checkout("${hashToCheckout}", 'simorgh')
                     Simorgh.installNodeModules()
                     stage ('Live E2Es') {
+                        echo "e2e's running againt commit: ${hashToCheckout}"
                         Simorgh.runE2Es('live', false)
                     }
                 } finally {
+                    echo "completed"
```

Comment:
Minor quibble but is this message potentially misleading as the tests might not have run?

Reply:
ahh, was a left over debug statement